app/src/main/java/org/p2p/wallet/auth/interactor/restore/UserRestoreInteractor.kt

```diff
@@ -50,12 +53,13 @@ class UserRestoreInteractor(
                 thirdShare = customShare,
                 encryptedMnemonic = encryptedMnemonic
             )
+
             signUpDetailsStorage.save(
                 data = Web3AuthSignUpResponse(
                     ethereumPublicKey = result.ethereumPublicKey,
                     mnemonicPhrase = result.mnemonicPhrase,
                     encryptedMnemonicPhrase = JsonObject(),
-                    deviceShare = null,
+                    deviceShare = deviceShare,
```

Comment:
why we are setting device share if we are doing SocialPlusCustomShare?

Reply:
cause if user has device share on device, we should not clear it. Why ? I don't know)